lib/commands/info/index.js

```diff
@@ -90,15 +91,24 @@ function setupCommand (name, description, argv, importMeta) {
 
   const versionSeparator = rawPkgName.lastIndexOf('@')
 
+  let pkgVersion, pkgName
   if (versionSeparator < 1) {
-    throw new InputError('Need to specify a full package identifier, like eg: webtorrent@1.0.0')
-  }
-
-  const pkgName = rawPkgName.slice(0, versionSeparator)
-  const pkgVersion = rawPkgName.slice(versionSeparator + 1)
-
-  if (!pkgVersion) {
-    throw new InputError('Need to specify a version, like eg: webtorrent@1.0.0')
+    // Get the latest version
+    try {
+      pkgName = rawPkgName
+      const response = await fetch(`https://registry.npmjs.org/${rawPkgName}/latest`)
+      /** @type any */
```

Comment:
Not a fan of using `any` but it seems like there's [an open discussion](https://github.com/node-fetch/node-fetch/issues/1262) on the node-fetch repo about it.